### PR TITLE
Wifi configuration on build & Armbian freeze

### PR DESCRIPTION
**Pull request by Stadicus**
Various minor fixes and features adds
* freeze Armbian at latest stable release (sunxi-4.20)
* build option to manually set root password (development only)
* build option to configure default wifi network (needs additional hardware module)
* display build options (dedicated script function)

### CODE REVIEW

File: armbian/base/build/customize-armbian-rockpro64.sh [hunk 1]
```diff
@@ -740,6 +758,22 @@ DNSSEC=yes
 Cache=yes
 EOF
 
+cat << EOF > /etc/network/interfaces
```
Comment (hkjn) [inline armbian/base/build/customize-armbian-rockpro64.sh]: Do we need to override `/etc/network/interfaces`? On the `eve` node, built from a recent commit on `master`, it seems to already exist and have the equivalent content as the file you're creating here:

```
root@eve:~# cat /etc/network/interfaces
source /etc/network/interfaces.d/*
# Network is managed by Network manager
auto lo
iface lo inet loopback
```

Comment (Stadicus) [reply armbian/base/build/customize-armbian-rockpro64.sh]: I'll try if it just works like that.

Comment (Stadicus) [reply armbian/base/build/customize-armbian-rockpro64.sh]: Removed the explicit `/etc/network/interfaces`, works just fine.

**Comment by hkjn** [comment]
Post-merge utACK 84a201063221bb340b73ad36e6d2bcc9108fad52

**Pull request merged**